supervisor/shared/web_workflow/static/serial.js

```diff
@@ -43,6 +43,8 @@ ws.onmessage = function(e) {
   } else if (e.data == "\x1b[K") { // Clear line
     log.textContent = log.textContent.slice(0, -left_count);
     left_count = 0;
+  } else if (e.data == "\x1b[2K\x1b[0G") {
```

Comment:
It might be better to instead ignore every escape sequence.

This could be done really fast with ignoring everything from when we see "ESC[" to whenever we see an alphabet letter.
(65-90 or 97-122, the alphabet letter should be discarded too)